lib/paperclip_processors/cropper.rb

```diff
@@ -0,0 +1,19 @@
+module Paperclip
+  class Cropper < Thumbnail
+    def transformation_command
+      if crop_command
+        crop_command + super.join(' ').sub(/ -crop \S+/, '').split(' ')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.